### PR TITLE
Update large tests to run with docker-compose

### PR DESCRIPTION
**Pull request by kjlyon**
Summary of changes:
- Updates for pluginsync v0.1.13
- Added large to test matrix
- Added docker-compose.yml file

Testing done:
- large tests
- small tests

image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
   - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
+  - GLIDE_HOME="${HOME}/.glide"
   matrix:
   - TEST_TYPE=small
+  - TEST_TYPE=large
   - TEST_TYPE=build
 matrix:
   exclude:
-  - go: 1.6.3
+  - go: 1.6.x
```
Comment (nanliu) [inline .travis.yml]: We should also exclude 1.6.x from the large test. We only need large test on 1.7.x

**Comment by nanliu** [comment]
We need a yml example task otherwise the large test just verifies the plugin loads, and will not load any tasks.

**Comment by kjlyon** [comment]
I updated to address your comments

**Comment by nanliu** [comment]
LGTM, please squash and feel free to merge.

**Pull request merged**